src/templates/speciality.js

```diff
@@ -94,14 +746,19 @@ const Speciality = ({
       />
       {tutorials && tutorials.length ? (
         <TutorialsSection speciality={title} tutorials={tutorials} />
-      ) : null}
+      ) : (
+        undefined
+      )}
```

Comment:
Everywhere else in this return we're doing `x && x.length && Component` - let's do the same here

Reply:
Actually, I have a feeling why that was like that - sometimes (not sure exactly why) if you use the array && array.length && <Component> approach, the logic will end up rendering literal 0 on the website in place of the component (I've had this issue). I'll change it for consistency.

Reply:
I was right (see below!). I used `{!!(tutorials && tutorials.length) &&
        <TutorialsSection speciality={title} tutorials={tutorials} />
      }` instead. 

![image](https://user-images.githubusercontent.com/31079643/60524854-37419700-9ce5-11e9-8395-2e7e68b90c9f.png)

Reply:
`talks.length > 0` will prevent that